awx_collection/test/awx/test_group.py

```diff
@@ -26,9 +26,8 @@ def test_create_group(run_module, admin_user):
     result.pop('invocation')
     assert result == {
         'id': group.id,
-        'group': 'Test Group',
+        'name': 'Test Group',
```

Comment:
Why is "credential_type" provided for backward compatibility, but "group" is not?

I'm not against dropping support for a returned key, but it needs to be declared in the README.md with the other backward-incompatible changes, and one way or the other, these should be consistent about whether they kept the old ones or got rid of them.